src/test/regress/expected/with_join.out

```diff
@@ -168,9 +167,15 @@ ORDER BY
   1,2,3
 LIMIT
   5;
-ERROR:  cannot pushdown the subquery
-DETAIL:  Complex subqueries and CTEs cannot be in the outer part of the outer join
-SET citus.enable_cte_inlining TO true;
+ user_id |              time               | event_type
```

Comment:
I think this query should fail, this is to show intermediate_result LEFT JOIN table is no supported. hence materialized cte

Reply:
all for the tests iin this file